source/plugin/offline-userplayer-data.rst

```diff
@@ -0,0 +1,79 @@
+=======================================
+Accessing data of offline Users/Players 
+=======================================
+
+In almost every plugin it's necessary to get access to player information even when the player is offline.
```

Comment:
I would use a less generalizing statement here. Most plugins may not need to use any kind of offline player data, so it would be better to say something along the lines of "It may be necessary for plugins to access..."